systemvm/noVNC/dashboard.css

```diff
@@ -0,0 +1,20 @@
+.vnc-controls {
```

Comment:
For all the assets/files in `systemvm/noVNC` add the apache license 2.0 in the header, see other js/css files for reference.

Reply:
added apache license to file I have created. rest files belong to noVNC